core/domain/feedback_domain.py

```diff
@@ -19,11 +19,30 @@
 
 
 class FeedbackThread(object):
-    """Domain object for a feedback thread."""
+    """Domain object for a feedback thread.
+    
+    Attributes:
+        full_thread_id: str. The feedback thread ID.
+        exploration_id: str. The associated exploration's ID.
+        state_name: str. The name of the state associated with 
+            the feedback thread.
+        original_author_id: str. The ID of the original author.
+        status: string(select). The current status of the 
+            thread.
+        subject: str. The subject of the feedback thread.
+        summary: str. A summary of the feedback thread.
+        has_suggestion: bool. Indicates whether a given exploration has a 
+            change suggestion.
```

Comment:
Well, I think `a suggestion to make a change` makes more sense to me. But English is not my native language, so maybe `a change suggestion` or `a change recommendation` is clear enough.